src/combinatorics/simple_graph/basic.lean

```diff
@@ -344,6 +386,25 @@ by { ext v w, simp only [from_edge_set_adj, set.mem_empty_iff_false, false_and,
 @[simp] lemma from_edge_set_univ : from_edge_set (set.univ : set (sym2 V)) = ⊤ :=
 by { ext v w, simp only [from_edge_set_adj, set.mem_univ, true_and, top_adj] }
 
+@[simp] lemma from_edge_set_inf (s t : set (sym2 V)) :
+  from_edge_set (s ∩ t) = from_edge_set s ⊓ from_edge_set t :=
```

Comment:
I suspect this and the following two lemmas will be better as `simp` lemmas if they're stated in in reverse (so simp normal form is to coalesce multiple `from_edge_set`s into a single `from_edge_set`). That way the edge sets have a better shot at interacting with each other.